### ISSUE TITLE
Template Files - Toolbar buttons missing

### ISSUE THREAD

**Issue by jonaskohl** [RealRaven2000/SmartTemplate4#143]
When trying to add a template file, the toolbar buttons for adding, removing or moving a template file are missing. (See attached screenshot)
![Screenshot 2021-08-11 140749](https://user-images.githubusercontent.com/23175424/129025901-cc0fae05-6f53-41ed-8d47-ac27459b95e4.png)

**Comment by RealRaven2000** [comment]
OK, looks weird - what version of Thunderbird and SmartTemplates are you using? What OS?
What happens if you try to increase the window height by dragging down on the bottom right corner?

**Comment by jonaskohl** [comment]
I'm using Thunderbird 78.13.0 on Windows 10 2004 and SmartTemplates 3.6.

It looks the same when dragging down the window.

![Screenshot 2021-08-11 171955](https://user-images.githubusercontent.com/23175424/129057693-9ab32b97-c7cc-4da5-b2a3-87d00ff68948.png)

After a bit of digging with the DevTools, I found this CSS rule (added at `xul.css:54` by ThunderBird apparently):

```css
*|*:root[chromehidden~="menubar"] .chromeclass-menubar, *|*:root[chromehidden~="directories"] .chromeclass-directories, *|*:root[chromehidden~="status"] .chromeclass-status, *|*:root[chromehidden~="extrachrome"] .chromeclass-extrachrome, *|*:root[chromehidden~="location"] .chromeclass-location, *|*:root[chromehidden~="location"][chromehidden~="toolbar"] .chromeclass-toolbar, *|*:root[chromehidden~="toolbar"] .chromeclass-toolbar-additional {
    display: none;
}
```

So after adding this CSS rule:

```css
.templateList > toolbarbutton {
    display: inline;
}
```

the buttons once again display as normal.

![Screenshot 2021-08-11 172517](https://user-images.githubusercontent.com/23175424/129057730-0f44a029-b29b-4b08-971c-eb436c355764.png)

So I think this has to be added to the add-on's CSS file for it to fix it.

**Comment by RealRaven2000** [comment]
So I haven'#t see this rule here on Windows 7; I would believe it is this part
 *|*:root[chromehidden~="toolbar"] .chromeclass-toolbar-additional {
    display: none;
}

as the buttons have the class "chromeclass-toolbar-additional". I wonder if this is specific to Windows 10, that would be a bad bug.

**Comment by RealRaven2000** [comment]
I also tested on my recording PC / DAW which is win10 64bit but the toolbar was visible. Probably no harm in adding the rule anyway. The display value for the buttons here reads as "-moz-box"...

**Comment by RealRaven2000** [comment]
I added

```
#fileTemplateToolbar.templateList > toolbarbutton.chromeclass-toolbar-additional {
  display: inline-block;
}
```

I wasn't quite sure whether the descendant selector added enough specificity. Can you check this version?

[smartTemplate-fx-3.7pre25.zip](https://github.com/RealRaven2000/SmartTemplate4/files/6993887/smartTemplate-fx-3.7pre25.zip)

(drag into Add-ons manager after downloading, no need to unpack)

**Comment by RealRaven2000** [comment]
Fixed in 3.7 published 20/08/2021
